apps/site/assets/webpack.config.base.js

```diff
@@ -50,6 +50,19 @@ module.exports = {
           loader: "babel-loader"
         }
       },
+      {
+        // https://docs.react-async.com/getting-started/installation#transpiling-for-legacy-browsers
+        test: /\/node_modules\/react-async\//,
```

Comment:
The linked page doesn't really explain why, but it directs users to exclude `core-js` from transpilation. Did you try that / could that help here?

Reply:
The example given assumes you're using a single rule to compile your whole app, and want to transpile _all_ of your `node_modules` instead of just react-async — in this case it's necessary to exclude the `core-js` package. Here we're creating a separate rule that only includes react-async, so we don't have to worry about that. But, as a separate tech debt task we should investigate whether we want to transpile all our packages. If nothing else it would solve this "oh I didn't realize it wouldn't work in IE" problem for good.